linkml/generators/common/ifabsent_processor.py

```diff
@@ -1,7 +1,13 @@
 import abc
 import re
+import sys
 from abc import ABC
-from typing import Any, Optional
+from typing import Any, Optional, Type, Union
+
+if sys.version_info.minor < 10:
```

Comment:
While looking up some information about the changes in Python 3.10's typings, I found that it could be interesting to also check the major version to avoid "issues" in the future if Python 4 comes out as Python 2 only goes to minor version 7 😇 

```suggestion
if sys.version_info < (3, 10):
```

I promise that my nitpicking self was under control but it's getting harder and harder 🤣

Reply:
(You can ignore this if you want, it's not critical for the next 5 years or so 😉 )

Reply:
reasonable ;)